docs/proposals/use-cascading-deletion/README.md

```diff
@@ -0,0 +1,388 @@
+---
+title: Use Cascading Deletion in Karmada
```

Comment:
How about naming this feature as `Migration Rollback Protection`? 

Forget `cascading` to avoid misleading.

Reply:
+1